web-registry/public/_redirects

```diff
@@ -1,5 +1,2 @@
+https://registry.regen.network/*  https://app.regen.network/:splat 301!
 /*    /index.html   200
-
-/projects  /projects/wilmot  200
-/projects/impactag  /projects/wilmot  200
```

Comment:
removed because this rule would not ever match, because the rule that was before it always matches (/* /index.html 200)